### ISSUE TITLE
기존 연구 결과 정리

### ISSUE THREAD

**Issue by chatterboy** [chatterboy/fracture#30]
## 목적

딥러닝을 이용한 의료 영상에 대한 골절 예측

### 데이터

- 골절 부위를 포함한 영상 데이터
- 각 환자에 대한 CT와 X-ray 영상으로 구성
- X-ray 영상은 CT 영상에 비해 10%

### 문제정의

- 각 영상은 1개 이상의 클래스를 포함
- multi-label classification + deep learning

### 데이터셋

- CT 영상만 고려
- 모든 CT 영상은 동일한 차원을 가짐
- 학습 데이터는 70%, 테스트 데이터는 30%로 구분

### 모델정의

- `resnet-50` from scratch
- 각 출력 노드에 `sigmoid`를 적용함으로써, multi-label classification 문제 접근

### 전처리

- mean subtraction 적용

### Data augmentation

- scaling, flipping, rotating, translating과 같은 네 가지 방법을 적용
- 학습 데이터의 수가 1,080배 증가

### 실험방법

아래와 같이 설정

```
num_classes = 12
batch_size = 64
height = 224
width = 224
num_channels = 3
learning_rate = 0.001
num_epochs = 3000
test_cycles = 500
```

### 실험결과

epoch | accuracy | loss
-|-|-
0 | 0 | 4.458
500 | 0.515 | 0.153
1000 | 0.765 | 0.084
1500 | 0.75 | 0.055
2000 | 0.734 | 0.079
2500 | 0.781 | 0.066

## 방향

- 클래스 구분에 대한 문제
- Data augmentation 방법 추가 고려
- 더 효율적인 multi-label classification + deep learning 방법 고려
- CT와 X-ray 영상을 모두 고려
- 족관절 골관절염 관련 deep learning 모델 프로세스 고려? (major voting ensemble + XGBoosting)
